resources/assets/lib/chat/chat-orchestrator.ts

```diff
@@ -74,33 +74,29 @@ export default class ChatOrchestrator implements DispatchListener {
     });
   }
 
-  focusNextChannel() {
-    const channelStore = this.rootDataStore.channelStore;
-    const channel = channelStore.get(this.rootDataStore.uiState.chat.selected);
-    if (channel && (channel.exists || channel.newChannel)) {
-      return;
+  focusChannelAtIndex(index: number) {
+    const channelList = this.rootDataStore.channelStore.channelList;
+    if (channelList.length === 0) {
+      return this.rootDataStore.channelStore.loaded = false;
```

Comment:
is it actually intended to return `false` or just for line saving :thinking:

Reply:
line saving 😔